llvm/test/CodeGen/SPARC/inlineasm-bad.ll

```diff
@@ -11,3 +11,12 @@ entry:
   tail call void asm sideeffect "faddq $0,$1,$2", "{f38},{f0},{f0}"(fp128 0xL0, fp128 0xL0, fp128 0xL0)
   ret void
 }
+
+; CHECK-label:test_twinword_error
```

Comment:
Bad formatting (note capitalisation, spacing and trailing colon for the label):

```CHECK-LABEL: test_twinword_error:```

Repeated in inlineasm.ll, where you can see the mismatch with the surrounding code.